cgs_vmc/utils.py

```diff
@@ -83,6 +83,7 @@ def create_hparams(**kwargs: Any) -> tf.contrib.training.HParams:
       basis_file_path='',
 
       wavefunction_type='',
+      composite_wavefunction_types=['', ''],
```

Comment:
Make this a tuple, rather than a list. Tuples are immutable and will complain if you are trying to append elements or set more than needed.

Also, please add the description of this parameter in the docstring something along these lines:
```A tuple of strings specifying individual types of the composite wavefunction. e.g. To crease a sum of conv_1d and fully_connected wavefunction use wavefunction_type='composite'; composite_wavefunction_types=('conv_1d', 'fully_connected').```